arbor/include/arbor/mechanism_abi.h

```diff
@@ -5,6 +5,9 @@
 
 #ifdef __cplusplus
 extern "C" {
+
+using namespace arb;
```

Comment:
I did a brief test test and the `using namespace` statement seems to be global if placed within an `extern` block, which does not limit its scope. So you might have to use the namespace prefix instead, since this is a header file.

Reply:
Or alternatively, I believe an enclosing `namespace arb { ... }` around the extern block should not affect any C symbol names inside and allow you to omit the prefix.

Reply:
That's an interesting interaction! I should have trusted my former self here.

Reply:
Reverted.